### PR TITLE
First feature

### PR DESCRIPTION
**Pull request by Lal2023**
New pull request

### CODE REVIEW

**Comment by Rowdy2018** [comment]
Looks good

**Review by Rowdy2018 — DISMISSED** [review]
Change the configuration as shown in the comments

File: first [hunk 1]
```diff
@@ -2,9 +2,5 @@
 
 {config: input=1, output=2}
 
-{Feature: first-feature}
+{Feature: first}
```
Comment (Rowdy2018) [inline first:5]: Add another line to set priority

{Priority: moderate}

**Review by Rowdy2018 — DISMISSED** [review]

File: first (hunk 1 above)
Comment (Lal2023) [reply first:5]: Tested the change. Works fine

Comment (Rowdy2018) [reply first:5]: add the additional line also.

File: first [hunk 2]
```diff
@@ -1,3 +1,8 @@
 #This is a test file
 
 {config: input=1, output=2}
+
+{Feature: first}
+
+{Priority: moderate}
+
```
Comment (Lal2023) [inline first:8]: Resolve this too

Comment (Rowdy2018) [reply first:8]: Works fine

**Review by Rowdy2018 — APPROVED** [review]
All clear. Merge this

**Comment by Lal2023** [comment]
Merging all

**Pull request merged**